packages/terra-slide-panel-manager/src/terra-dev-site/test/slide-panel-manager/SlidePanelManagerDefault.test.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
Since we are doing these for internal test files I feel we can have single scss file for all the tests unless there are two many classes needs to be added for component.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.